src/components/NotesContentEditor/NotesContentEditorStyles.js

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 export const StyledSelectedNotesContent = styled.div`
   position: absolute;
   z-index: 1;
-  top: 0;
+  top: 156px;
```

Comment:
Where does this number come from? :)

I wouldn't say it looks right...

![image](https://user-images.githubusercontent.com/6686679/82831384-f6380f80-9ec0-11ea-80ac-838f3c5afe84.png)

Guess it should be `56px`? :)

Let's create a variable `topBarHeight` ? :)

`56px` is also used in `NotesMindMapStyles.js` on line 6.

Reply:
And `56px` is also sued for `StyledTopBar` height in `TopBarStyles.js`, line 17 :)

Reply:
The NotesMindMap was moved more than on 56px, because I want to show contrast of the bottom line of TopBar. This is temporary styles.

Reply:
I'll change that to `top: 65px`. Is it good? So we will have nicer look.

Reply:
It's will be look like that
![image](https://user-images.githubusercontent.com/19979138/82926739-4f1eab00-9f88-11ea-9cfd-14bb1206c420.png)

Reply:
Sure 👍